GVRf/Framework/framework/src/main/res/raw/vertex_template_multitex.vsh

```diff
@@ -123,25 +127,35 @@ void main() {
 // This section contains code to compute
 // vertex contributions to lighting.
 //
-	LightVertex(vertex);
+    LightVertex(vertex);
 #endif
+
 #ifdef HAS_TEXCOORDS
 //
 // This section contains assignment statements from
 // input vertex attributes to output shader variables
 // generate by GVRShaderTemplate during shader construction.
 //
-	@TEXCOORDS
+@TEXCOORDS
+
 #endif
 	viewspace_position = vertex.viewspace_position;
 	viewspace_normal = vertex.viewspace_normal;
 	view_direction = vertex.view_direction;
 #ifdef HAS_MULTIVIEW
+<<<<<<< HEAD
 	 bool render_mask = (u_render_mask & (gl_ViewID_OVR + uint(1))) > uint(0) ? true : false;
      mat4 mvp = u_mvp_[gl_ViewID_OVR];
      if(!render_mask)
          mvp = mat4(0.0);  //  if render_mask is not set for particular eye, dont render that object
      gl_Position = mvp  * vertex.local_position;
+=======
```

Comment:
Looks like unresolved merge conflict.

Reply:
You are right but it does not appear in my version! I will try to fix it. Maybe I can do a force push...